src/cdk/a11y/_a11y.import.scss

```diff
@@ -0,0 +1,2 @@
+@forward 'a11y' hide a11y, high-contrast;
```

Comment:
What happens if we remove this top line? My guess as to what's happening here is that it's attempting to forward `cdk-optionally-nest-content`, but that then gets hidden on the following line.

Reply:
I tried something like this locally:
```scss
// parent.scss
@mixin one() { .one { color: red; } }
@mixin two() { .two { background: blue; } }
@mixin three() { .three { outline: yellow;} }
```

```scss
// child.scss
@forward 'parent' as num-* hide three;
```

```scss
// leaf.scss
@import 'child';
@include one();
```

Running `sass ./leaf.scss` errors here with `Error: Undefined mixin.` since the mixin is aliased as part of the `@forward`. Including `num-one()` instead works fine. I think that without explicitly forwarding the mixins, they aren't surfaced?